osu.Game/Online/Multiplayer/MultiplayerRoom.cs

```diff
@@ -39,14 +39,17 @@ public class MultiplayerRoom
         /// All users currently in this room.
         /// </summary>
         [Key(3)]
-        public List<MultiplayerRoomUser> Users { get; set; } = new List<MultiplayerRoomUser>();
+        public IList<MultiplayerRoomUser> Users { get; set; } = new List<MultiplayerRoomUser>();
```

Comment:
Is there a reason for this change?

Reply:
This is used server-side (see `ServerMultiplayerRoom`).

Reply:
Yikes! Just to handle user joins/parts? That seems very overengineered.

Reply:
It was the simplest way I could think of, but I'm definitely open to alternatives (I brought this up in conversation on discord as I wasn't sure about it).

Reply:
I guess this isn't a breaking change so can be reverted at any point, but I'll comment on the server-side usages.